### ISSUE TITLE
Bug in decreasing variable prices by percentage

### ISSUE THREAD

**Issue by jcsnssafety** [woocommerce/woocommerce#12424]
## EXPLANATION OF THE ISSUE

Whenever several product variations are changed using the drop down menu to decrease by perscentage, the result is widely inaccurate.

5.65 when reduced by 17.5% should be 4.8085106 but Woocommerce returns 4.66125

## STEPS TO REPRODUCE THE ISSUE

Choose a product that has variations
Under Product Data - Variations choose from the dropdown box choose Decrease Regular Prices (fixed amount or percentages) and press Go
Type in 17.5%
Press Okay

## SYSTEM STATUS REPORT

```
Grab the system status report from WooCommerce > System Status and paste it here.
```
`
### WordPress Environment ###

Home URL: http://reliancemedical.showmemysite.co.uk
Site URL: http://reliancemedical.showmemysite.co.uk
WC Version: 2.6.8
Log Directory Writable: ✔
WP Version: 4.6.1
WP Multisite: –
WP Memory Limit: 320 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_GB

### Server Environment ###

Server Info: Apache/2.4.23 (Unix) OpenSSL/1.0.2h mod_bwlimited/1.4 PHP/5.6.28
PHP Version: 5.6.28
PHP Post Max Size: 64 MB
PHP Time Limit: 3000
PHP Max Input Vars: 1000
cURL Version: 7.38.0
OpenSSL/1.0.2h

SUHOSIN Installed: –
MySQL Version: 5.6.34
Max Upload Size: 64 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 2.6.8
: 
woocommerce_sessions: ✔
woocommerce_api_keys: ✔
woocommerce_attribute_taxonomies: ✔
woocommerce_downloadable_product_permissions: ✔
woocommerce_order_items: ✔
woocommerce_order_itemmeta: ✔
woocommerce_tax_rates: ✔
woocommerce_tax_rate_locations: ✔
woocommerce_shipping_zones: ✔
woocommerce_shipping_zone_locations: ✔
woocommerce_shipping_zone_methods: ✔
woocommerce_payment_tokens: ✔
woocommerce_payment_tokenmeta: ✔
MaxMind GeoIP Database: ✔

### Active Plugins (1) ###

WooCommerce: by WooThemes – 2.6.8

### Settings ###

Force SSL: –
Currency: GBP (£)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2

### API ###

API Enabled: ✔

### WC Pages ###

Shop Base: #6 - /shop/
Basket: #7 - /cart/
Checkout: #1742 - /checkout/
My Account: #2727 - /my-account-2/

### Taxonomies ###

Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

### Theme ###

Name: Twenty Fourteen
Version: 1.8
Author URL: https://wordpress.org/
Child Theme: ❌ – If you're modifying WooCommerce on a parent theme you didn't build personally
then we recommend using a child theme. See: How to create a child theme

WooCommerce Support: ✔

### Templates ###

Overrides: –
`

**Comment by mikejolley** [comment]
@claudiosanches this is broken in master (no results) FYI

**Comment by mikejolley** [comment]
This is now fixed in 2.7 (master).
